sql/src/test/scala/org/datasyslab/geosparksql/predicateTestScala.scala

```diff
@@ -60,5 +60,15 @@ class predicateTestScala extends TestBaseScala {
       resultDf.show()
       assert(resultDf.count() == 999)
     }
+    it("Passed ST_Touches") {
+      var pointCsvDF = sparkSession.read.format("csv").option("delimiter", ",").option("header", "false").load(csvPointInputLocation)
+      pointCsvDF.createOrReplaceTempView("pointtable")
+      var pointDf = sparkSession.sql("select ST_Point(cast(pointtable._c0 as Decimal(24,20)), cast(pointtable._c1 as Decimal(24,20))) as arealandmark from pointtable")
+      pointDf.createOrReplaceTempView("pointdf")
+
+      var resultDf = sparkSession.sql("select * from pointdf where ST_Touches(pointdf.arealandmark, ST_PolygonFromEnvelope(0.0,99.0,1.1,101.1))")
+      resultDf.show()
+      assert(resultDf.count() == 1)
```

Comment:
Thanks for your contribution!

This assertion should check the ST_Touches correctness which is "true" or "false" instead of checking DF count. Please add a "true" test case and a "false" test case.

Thanks!

Reply:
@Lancher Sorry, I misunderstood your code. Forget about my comment. Just solve @netanel246 's comment - change the version to 1.2.0.

Thank you!